tools/toolchains/mbed_toolchain.py

```diff
@@ -109,6 +109,8 @@
                         "__MBED_CMSIS_RTOS_CM", "__DSP_PRESENT=1U"],
 }
 
+UNSUPPORTED_OPTION_SMALL_EXECPTION_STRING = 'small C library option not supported for this target.'
```

Comment:
You only need one variable. Try this.
```suggestion
UNSUPPORTED_DEFAULT_LIB_EXECPTION_STRING = "{} C library option not supported for this target."
```